helm/chart-operator-chart/values.yaml

```diff
@@ -23,5 +26,5 @@ resources:
     cpu: 250m
     memory: 250Mi
 
-cnr:
-  address: https://quay.io
+tiller:
+  namespace: kube-system
```

Comment:
Defaults to kube-system and we override to giantswarm when managed by cluster-operator.